<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Asus-Vivobook14" failures="0" tests="5" name="workingWithtTestNG.DependencyOnMethods" time="0.018" errors="1" timestamp="2023-01-03T11:15:36 IST" skipped="0">
  <testcase classname="workingWithtTestNG.DependencyOnMethods" name="modify" time="0.001"/>
  <system-out>
modify method  </system-out>
  <testcase classname="workingWithtTestNG.DependencyOnMethods" name="delete" time="0.004">
    <error message="/ by zero" type="java.lang.ArithmeticException">
      <![CDATA[java.lang.ArithmeticException: / by zero
at workingWithtTestNG.DependencyOnMethods.delete(DependencyOnMethods.java:30)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- delete -->
  <system-out/>
  <testcase classname="workingWithtTestNG.DependencyOnMethods" name="update" time="0.001"/>
  <system-out>
update method  </system-out>
  <testcase classname="workingWithtTestNG.DependencyOnMethods" name="create" time="0.001"/>
  <system-out>
create method  </system-out>
  <testcase classname="workingWithtTestNG.DependencyOnMethods" name="register" time="0.011"/>
  <system-out>
register method  </system-out>
</testsuite> <!-- workingWithtTestNG.DependencyOnMethods -->
